src/lib/apexPmd.ts

```diff
@@ -137,9 +146,16 @@ export class ApexPmd {
         const targetPathKey = `-d "${targetPath}"`;
         const rulesetsKey = `-R "${rulesetsArg}"`;
 
-        const pmdKeys = `${formatKey} ${cacheKey} ${targetPathKey} ${rulesetsKey}`
+        const pmdKeys = `${formatKey} ${cacheKey} ${targetPathKey} ${rulesetsKey}`;
+
+        const classPath = [
+            path.join(this._pmdPath, 'lib', '*'),
+            path.join(vscode.workspace.rootPath,'*'),
+            ...this._additionalClassPaths
+        ].join(CLASSPATH_DELM);
+
+        const cmd = `java -cp '${classPath}' net.sourceforge.pmd.PMD ${pmdKeys}`;
```

Comment:
To work on windows, the arg delimiters wrapped around ${classpath} need to be double quotes, not single ones. I note that rulesetsKey and targetPathKey already use double quotes, so presumably this will also work outside Windows too?

Reply:
@andrewgilbertsagecom good catch... Updated

Reply:
@ChuckJonas Works a treat, thank you very much!